### PR TITLE
10466 Design Debt

### PR DESCRIPTION
**Pull request by akuny**

### CODE REVIEW

**Review by zachrog — APPROVED** [review]

File: shared/src/business/entities/NewPetitionerUser.test.ts [hunk 1]
```diff
@@ -17,17 +17,42 @@ describe('NewPetitionerUser', () => {
...
+      expect(formEntity.getValidationErrors()).toEqual({
+        email: 'Enter email address in format: yourname@example.com',
+      });
+    });
+
+    it('should return an error message for email when an email address exceeding the maximum number of characters is provided', () => {
+      const formEntity = new NewPetitionerUser({
+        ...validEntity,
+        email:
+          'horatio_says_tis_but_our_fantasy_and_will_not_let_belief_take_hold_of_him_touching_this_dreaded_sight_twice_seen_of_us@denmarkemailservice.com',
```
Comment (Mwindo) [inline shared/src/business/entities/NewPetitionerUser.test.ts:48]: This singlehandedly improves our codebase by six points.

File: web-client/src/presenter/computeds/CreatePetitionerAccount/createAccountHelper.test.ts [hunk 2]
```diff
@@ -79,6 +79,20 @@ describe('createAccountHelper', () => {
         },
       });
 
+      expect(result).toMatchObject({
+        email: 'Enter email address in format: yourname@example.com',
+      });
+    });
+
+    it('should return the correct error message when an invalid email address is provided', () => {
```
Comment (Mwindo) [inline web-client/src/presenter/computeds/CreatePetitionerAccount/createAccountHelper.test.ts]: Can we give this a different test name than above?

File: shared/src/business/entities/NewPetitionerUser.ts [hunk 3]
```diff
@@ -25,8 +25,9 @@ export class NewPetitionerUser extends JoiValidationEntity {
       .messages({ '*': 'Passwords must match' }),
     email: JoiValidationConstants.EMAIL.required()
       .messages({
-        '*': 'Enter a valid email address',
-        'string.max': 'Email address must contain fewer than 100 characters', //todo test
```
Comment (Mwindo) [inline shared/src/business/entities/NewPetitionerUser.ts:29]: Not really anything to do with this PR, but is there a reason we don't enforce character limits for other email validations?

Comment (Mwindo) [reply shared/src/business/entities/NewPetitionerUser.ts:29]: E.g., I wonder if it eventually makes sense to have a generic email validation/messages thing for emails across the system.

**Review by Mwindo — APPROVED** [review]
Left one comment about naming that might be worth addressing, but otherwise LGTM--thanks, @akuny!

**Pull request merged**